server/controller/src/main/java/ai/starwhale/mlops/datastore/ColumnSchemaDesc.java

```diff
@@ -66,11 +66,23 @@ public class ColumnSchemaDesc {
     private String pythonType;
 
     /**
-     * This field represents the type of elements in array. It is only used when type is ARRAY. The name field of
+     * This field represents the type of elements in the list. It is only used when type is LIST. The name field of
      * elementType is never used.
      */
     private ColumnSchemaDesc elementType;
 
+    /**
+     * This field represents the type of keys in the map. It is only used when type is MAP. The name field of
+     * keyType is never used.
+     */
+    private ColumnSchemaDesc keyType;
+
+    /**
+     * This field represents the type of values in map. It is only used when type is MAP. The name field of
+     * valueType is never used.
+     */
+    private ColumnSchemaDesc valueType;
```

Comment:
Add `MAP` in the comment of `private String type;`

Reply:
Done